src/lib/amd.h

```diff
@@ -38,6 +38,8 @@ enum amd_led_interfaces {
 	AMD_INTF_UNSET,
 	AMD_INTF_SGPIO,
 	AMD_INTF_IPMI,
+	/* The new way to control nvme led with less dependence */
```

Comment:
It would be nice to expand this comment to explain why there is less dependence.